docs/source/archive/gsoc/reports/2025/vulnerablecode_michael.rst

```diff
@@ -0,0 +1,224 @@
+
+.. figure:: https://private-user-images.githubusercontent.com/29122581/480717461-29806bc6-faf5-48c9-8632-608c23d96e83.gif?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTU4MTMzNTksIm5iZiI6MTc1NTgxMzA1OSwicGF0aCI6Ii8yOTEyMjU4MS80ODA3MTc0NjEtMjk4MDZiYzYtZmFmNS00OGM5LTg2MzItNjA4YzIzZDk2ZTgzLmdpZj9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA4MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwODIxVDIxNTA1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWUwOTRkNWI1OTViNzYxODM4MjAyYTBjYTdmY2QyMzQ1Mzg2MTVmM2M5N2Q0M2I1MDQwMGRiNWJjZDllNmRjODQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.lmRcQBxwP1kbhcQ8Siq6vvm1GrBfd_BhkIbIte2NuYs
+   :alt: Live evaluation demo in VulnTotal browser extension
+   :align: center
+   :width: 70%
+
+   VulnTotal and its browser extension consuming the new live evaluation API.
+
+Linked Pull Requests
+--------------------
+
+.. list-table::
+   :widths: 10 40 20
+   :header-rows: 1
+
+   * - Sr. no
+     - Name
+     - Link
+   * - 1
+     - Add Live Evaluation API endpoint and PyPa live pipeline importer
+     - `aboutcode-org/vulnerablecode#1969
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1969>`_
+   * - 2
+     - Add Gitlab Live V2 Importer
+     - `aboutcode-org/vulnerablecode#1910
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1910>`_
+   * - 3
+     - Add Curl Live Importer V2
+     - `aboutcode-org/vulnerablecode#1923
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1923>`_
+   * - 4
+     - Add Elixir Security Live V2 Importer
+     - `aboutcode-org/vulnerablecode#1935
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1935>`_
+   * - 5
+     - Add NPM Live Importer V2
+     - `aboutcode-org/vulnerablecode#1941
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1941>`_
+   * - 6
+     - Add GitHub OSV Live V2 Importer Pipeline
+     - `aboutcode-org/vulnerablecode#1977
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1977>`_
+   * - 7
+     - Add Postgres Live V2 Importer Pipeline
+     - `aboutcode-org/vulnerablecode#1982
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1982>`_
+   * - 8
+     - Add PySec Live V2 Importer Pipeline
+     - `aboutcode-org/vulnerablecode#1983
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1983>`_
+   * - 9
+     - Add Local VulnerableCode Datasource in VulnTotal and allow live evaluation
+     - `aboutcode-org/vulnerablecode#1985
+       <https://github.com/aboutcode-org/vulnerablecode/pull/1985>`_
+   * - 10
+     - Integrate Local VulnerableCode datasource and live evaluation
+     - `aboutcode-org/vulntotal-extension#17
+       <https://github.com/aboutcode-org/vulntotal-extension/pull/17>`_
+
+
+Closing Thoughts
+-------------------
+
+This project was an exciting step forward from my 2024 GSoC work. By moving
```

Comment:
Maybe link your previous GSoC 2024 page here, it's in the same sphinx docs so use https://www.sphinx-doc.org/en/master/usage/referencing.html#ref-role